### ISSUE TITLE
Folders with special characters in name won’t be skipped

### ISSUE THREAD

**Issue by supermueller** [bdkjones/CodeKit#358]
**Quick, short summary:**

If a folder with special characters in its name (e.g. a blank character) is be added to the Skipped Folders list, it will not be skipped.

**Expected results:**

The folders should be skipped correctly.

**Actual results:**

The “wrong“ named folders will not be skipped.

**Exact steps to reproduce:**

Create a folder called `Hello World`, in CodeKits file browser select and right-click on the folder, select “Skip This Folder“. The name of the folder appears in the Skipped Folder list, but is not grayed out in the files browser and containing files will still be processed.

**Comment by bdkjones** [comment]
Well...folders with spaces in their names aren't generally involved in websites because that space character would have to be percent-encoded in the URL. My advice would be to not use spaces in your folder names.

**Comment by supermueller** [comment]
You are right: Spaces in web projects aren’t a good idea – but in real world you sometimes have to use them. But if is to complicated to implement this feature, I can live with it.
